src/palace/manager/sqlalchemy/model/circulationevent.py

```diff
@@ -126,46 +122,3 @@ class CirculationEvent(Base):
 
     # The time format used when exporting to JSON.
     TIME_FORMAT = "%Y-%m-%dT%H:%M:%S+00:00"
-
-    @classmethod
```

Comment:
Since the local analytics provider now uses `AnalyticsEventData` to create its `CirculationEvent` objects, this method didn't really make sense. So I removed it, so it doesn't cause confusion in the future.